settings/language-sass.cson

```diff
@@ -2,13 +2,13 @@
   'editor':
     'autoIndentOnPaste': false
     'increaseIndentPattern': '''(?x)
-      ^\\s* # Start of line
+      ^(?!\\s*$)\\s* # Start of non empty line
       (
-        [^:]+ # No colons (for regular selectors like body)
+        [^:@,]+ # No colons, commas and at signs (for regular selectors like body)
```

Comment:
Yes, line 7 prevents at-rule to be qualified as regular selector (all regular selectors get indent). Line 11 detects proper at-rule and decides about its indentation.